Backend/nix/arion-configuration.nix

```diff
@@ -201,7 +201,7 @@ in
           description = ''
             Run pgadmin stack - Pgadmin in a docker container
           '';
-          args = "up --remove-orphans pg-admin";
+          args = "up -d --remove-orphans pg-admin";
```

Comment:
Please revert this Nix change. If you want to run detached mode, pass `-d` in the command line, like:

```
, run-monitoring -d
```